src/style.css

```diff
@@ -1 +1,116 @@
-/* add styles here */
+:root {
+  --custom-color: rgba(61, 78, 97, 1);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
```

Comment:
You should use a `@font-face` for all the fonts you'll use and link the .ttf file you have to it [Reference.](https://github.com/mate-academy/layout_search-bar-airbnb/tree/master#:~:text=use%20%40font%2Dface%20for%20fonts) Just like you did here you must also do for Avenir-Heavy